docs/src/pages/customization/components/components.md

```diff
@@ -232,133 +134,12 @@ Here are five alternatives; each has its pros and cons.
-
-### Theme variables
-
-You can adjust the [theme configuration variables](/customization/theming/#theme-configuration-variables).
-
-```jsx
-const theme = createMuiTheme({
-  typography: {
-    button: {
-      fontSize: '1rem',
-    },
-  },
-});
-```
-
-{{"demo": "pages/customization/components/ThemeVariables.js"}}
-
-### Global CSS override
-
-You can also customize all instances of a component with CSS.
-Components expose [global class names](/styles/advanced/#with-material-ui-core) to enable this.
-It's very similar to how you would customize Bootstrap.
-
-```jsx
-const GlobalCss = withStyles({
-  // @global is handled by jss-plugin-global.
-  '@global': {
-    // You should target [class*="MuiButton-root"] instead if you nest themes.
-    '.MuiButton-root': {
-      fontSize: '1rem',
-    },
-  },
-})(() => null);
-
-// …
-
-<GlobalCss />;
-```
-
-{{"demo": "pages/customization/components/GlobalCssOverride.js", "iframe": true, "height": 70}}
-
-### Global theme override
-
-You can take advantage of the `overrides` key of the `theme` to potentially change every single style injected by Material-UI into the DOM.
-Learn more about it in the [themes section](/customization/globals/#css) of the documentation.
-
-```jsx
-const theme = createMuiTheme({
-  components: {
-    MuiButton: {
-      styleOverrides: {
-        root: {
-          fontSize: '1rem',
-        },
-      },
-    },
-  },
-});
-```
-
-{{"demo": "pages/customization/components/GlobalThemeOverride.js"}}
-
-### Adding new component variants
-
-You can take advantage of the `variants` key in the `theme`'s components section to add new variants to Material-UI components. These new variants, can specify which styles the component should have, if specific props are defined together.
-
-The definitions are specified in an array, under the component's name. For every one of them a class is added in the head. The order is **important**, so make sure that the styles that should win will be specified lastly.
-
-```jsx
-const theme = createMuiTheme({
-  components: {
-    MuiButton: {
-      variants: [
-        {
-          props: { variant: 'dashed' },
-          style: {
-            textTransform: 'none',
-            border: `2px dashed grey${blue[500]}`,
-          },
-        },
-        {
-          props: { variant: 'dashed', color: 'secondary' },
-          style: {
-            border: `4px dashed ${red[500]}`,
-          },
-        },
-      ],
-    },
-  },
-});
-```
-
-If you are using TypeScript, you will need to specify your new variants/colors, using [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation).
-
-```tsx
-declare module '@material-ui/core/Button/Button' {
-  interface ButtonPropsVariantOverrides {
-    dashed: true;
-  }
-}
-```
-
-{{"demo": "pages/customization/components/GlobalThemeVariants.js"}}
+Please take a look on the [theme's global page](/customization/globals/) for more details.
```

Comment:
```suggestion
Please take a look at the theme's [global overrides page](/customization/globals/) for more details.
```

Reply:
We might want to update the page name 🙂

Reply:
I was thinking about the same thing, I believe that page should be customization/components, as it describes the `theme.components` options, but it would have a conflict with the name of the page I am updating now :D... So maybe `Global overrides` would be a better name..

Reply:
I'll do the renaming of the page in a separate PR to avoid too many changes here.